modular_skyrat/modules/reagent_forging/code/forge_weapons.dm

```diff
@@ -161,11 +147,19 @@
 	. = ..()
 	AddElement(/datum/element/kneejerk)
 
-/obj/item/forging/reagent_weapon/hammer/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
-	. = ..()
-	if(!is_type_in_list(target, fast_attacks))
+/obj/item/forging/reagent_weapon/hammer/attack(mob/living/target_mob, mob/living/user, params)
+	force = 19
+	return ..()
```

Comment:
This feels very much like a band-aid. Plus, the fact that it resets the variable like this isn't really good for maintainability, if you /really/ want this behavior done like this, try using `initial(force)` instead.